### PR TITLE
test(timeseries): add testcases to time series card

### PR DESCRIPTION
**Pull request by scottdickerson**
add testcases for timeseries card and dashboard header

### CODE REVIEW

File: src/components/ImageCard/ImageHotspots.jsx [hunk 1]
```diff
@@ -504,6 +504,7 @@ const ImageHotspots = ({
         <InlineLoading
           style={{ position: 'absolute', top: 0 }}
           description="Loading hotspot data..."
+          status="active"
```
Comment (tay1orjones) [inline src/components/ImageCard/ImageHotspots.jsx]: Constitutes a new patch version publish

File: src/components/ListCard/ListCard.jsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ const ListCard = ({ id, title, size, data, isLoading, loadData, hasMoreData, ...
                 })
               : null}
 
-            {isLoading ? <InlineLoading description="Loading data..." /> : null}
+            {isLoading ? <InlineLoading description="Loading data..." status="active" /> : null}
```
Comment (tay1orjones) [inline src/components/ListCard/ListCard.jsx]: Also constitutes a new patch version publish imo

File: src/components/TimeSeriesCard/TimeSeriesCard.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { useRef, useMemo, useCallback } from 'react';
 import moment from 'moment/min/moment-with-locales.min';
-import { LineChart } from '@carbon/charts-react';
+import LineChart from '@carbon/charts-react/line-chart';
```
Comment (tay1orjones) [inline src/components/TimeSeriesCard/TimeSeriesCard.jsx]: Does this just help tree shake the rest of the lib out?

Comment (scottdickerson) [reply src/components/TimeSeriesCard/TimeSeriesCard.jsx]: unfortunately this is required to trigger the transpilation of the linechart code, won't be needed once 0.16.10 comes out

File: yarn.lock [hunk 4]
```diff
@@ -4835,7 +4835,7 @@ capture-stack-trace@^1.0.0:
   version "1.0.1"
   resolved "https://registry.yarnpkg.com/capture-stack-trace/-/capture-stack-trace-1.0.1.tgz#a6c0bbe1f38f3aa0b92238ecb6ff42c344d4135d"
 
-carbon-components-react@^7.6.4:
+carbon-components-react@7.6.4:
```
[the remark this answers is not in the document]
Comment (scottdickerson) [reply yarn.lock]: fixed, thanks for catching

**Comment by tay1orjones** [comment]
@scottdickerson looks like coveralls is stuck. Any idea how to debug?

**Pull request merged**